en/02_Developer_Guides/08_Performance/00_Partial_Template_Caching.md

```diff
@@ -1,12 +1,16 @@
 ---
-title: Partial Caching
+title: Partial template caching
```

Comment:
I'll need to check to see if there are any existing links to this page that need to be updated, and I'm not sure if we need to manually add a redirect for this